picard/ui/itemviews/__init__.py

```
@@ -411,6 +411,16 @@ def update_colums_text(self, color=None, bgcolor=None):
             else:
                 if column.align == ColumnAlign.RIGHT:
                     self.setTextAlignment(i, QtCore.Qt.AlignmentFlag.AlignRight | QtCore.Qt.AlignmentFlag.AlignVCenter)
+                # Support custom columns with provider evaluation
+                try:
+                    from picard.ui.itemviews.custom_columns import CustomColumn  # Local import to avoid cycles
+
+                    if isinstance(column, CustomColumn):
+                        self.setText(i, column.provider.evaluate(self.obj))
+                        continue
+                except Exception:  # noqa: BLE001
```

Hiding exceptions isn't great, can you narrow matches at least? And/or log it (even at debug level).

---

Do not catch blind exceptions; logging: https://github.com/metabrainz/picard/pull/2714/commits/bfaa15e6e53b7e0eeeecdf49088d1df3d7359896